src/content/api/bank-accounts.mdx

```diff
@@ -0,0 +1,746 @@
+    When the Bank Account was created.
+  </Property>
+
+  <Property name="modifiedAt" type="timestamp">
+    When the Bank Account was updated..
+  </Property>
+
+  <Property name="createdBy" type="crn">
+    The User or API Key that created the Bank Account.
+  </Property>
+
+  <Property name="modifiedBy" type="crn">
+    The User or API Key that updated the Bank Account.
+  </Property>
+
+  <Property name="phoneNumber" type="string">
+    The user's phone number.
+  </Property>
+
+  <Property name="fullName" type="string">
+    The first and last name of the user.
+  </Property>
+
+  <Property name="emailAddress" type="string">
+    The user's email address.
+  </Property>
+
+  <Property name="test" type="boolean">
+    A flag which is present if the Bank Account is for testing.
+  </Property>
+</Properties>
+
+### Bank Account Type <Badge type="experimental"/>
+
+Types of bank accounts to allow access to different [Asset Types](https://docs.centrapay.com/api/asset-types).
+
+|   Type    |                          Description                           |
+| --------- | -------------------------------------------------------------- |
+| centrapay | Allows topup and withdrawal of the `centrapay.nzd` asset type. |
+| quartz    | Allows usage of the `quartz.nzd` asset type.                   |
+
+---
+
+## Bank Account Approval Type Summary Model <Badge type="experimental"/>
+
+A summary of the [Bank Account Approvals](https://docs.centrapay.com/api/bank-account-approvals) for a Bank Account.
+There is one object per type of Bank Account Approval, which provides a summary of the approval status.
+
+### Attributes
+<Properties>
+  <Property name="type" type="string">
+    The type of Bank Account Approval Summary.
+  </Property>
+
+  <Property name="status" type="string">
+    The summarized status of the Bank Account Approvals. Supported values are `pending`, `approved` and `declined`.
+  </Property>
+
+  <Property name="updatedAt" type="timestamp">
+    When the Bank Account Approval Summary was updated.
+  </Property>
+</Properties>
+
+---
+
+## Bank Account Balance Model <Badge type="experimental"/>
+
+The Bank Account balance, retrieved using Open Banking flows. The supported Bank Account type is `quartz`.
+
+### Attributes
+<Properties>
+  <Property name="bankAccountId" type="string">
+    The unique identifier of the Centrapay Bank Account.
+  </Property>
+
+  <Property name="balance" type="bignumber">
+    The Open Banking Bank Account balance.
+  </Property>
+
+  <Property name="currency" type="string">
+    	Currency code (eg. “NZD”).
+  </Property>
+</Properties>
+
+---
+
+<Endpoint
+  method="POST"
+  path="/api/bank-accounts"
+>
+  ## Create Bank Account
+
+  A Bank Account can be created with or without direct debit authorized.  By including
+  `directDebitAuthority`, the user accepts our [Direct Debit terms](https://centrapay.com/directdebit-termsandconditions/)
+  and has authority to operate this account.
```

Comment:
I think we have lost some context here about the optional fields. Maybe we need to add something like:

> All fields are required to create a direct-debit authority and are required for Top Up.